spec/unit/containerize_task_spec.rb

```diff
@@ -56,5 +57,16 @@
         to receive(:write_compose_file).with(workloads, output_path)
       containerize_task.containerize(system_description, output_path)
     end
+
+    it "shows the number of detected workloads" do
+      expect_any_instance_of(WorkloadMapper).
+        to receive(:identify_workloads).with(system_description).and_return(workloads)
+      expect_any_instance_of(WorkloadMapper).
+        to receive(:write_compose_file).with(workloads, output_path)
+      containerize_task.containerize(system_description, output_path)
+      expect(captured_machinery_output).to include(
+        "\nFound 1 workload. Wrote to"
+      )
+    end
```

Comment:
For the tests I would rather have one test, which shows how the output looks like and checks that the actual output matches that.